Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocListAsGridUserControl.ascx.cs

```diff
@@ -462,16 +467,7 @@ private void CreateValidationErrors ()
 
     static string FormatValidationFailure (ValidationFailure failure)
     {
-      return failure switch
-      {
-        PropertyValidationFailure propertyValidationFailure => $"Unhandled property validation failure '{propertyValidationFailure.ErrorMessage}'"
-                                                               + $" for property '{propertyValidationFailure.ValidatedProperty.Name}'"
-                                                               + $" on object '{propertyValidationFailure.ValidatedObject}'.",
-        ObjectValidationFailure objectValidationFailure => $"Unhandled object validation failure '{objectValidationFailure.ErrorMessage}'"
-                                                           + $" for properties '{string.Join(", ", objectValidationFailure.ValidatedProperties.Select(e => e.Property.Name))}'"
-                                                           + $" on object '{objectValidationFailure.ValidatedObject}'.",
-        _ => $"Unhandled property validation failure '{failure.ErrorMessage}' on object '{failure.ValidatedObject}'."
-      };
+      return $"Unhandled validation failure '{failure.ErrorMessage}' for property '{failure.ValidatedProperties.Single().Property.Name}' on object '{failure.ValidatedObject}'.";
```

Comment:
Message:
`$"Unhandled validation failure '{failure.ErrorMessage}' for properties 'FIRST', 'SECOND', 'THIRD', 'ETC' on object '{failure.ValidatedObject}'.";`
(please build via linq and string.join apis.